frontend/src/components/navbar/NavBar.js

```diff
@@ -25,21 +25,21 @@
       if(token) {
         return (
           <>
-           <nav className='nav-bar'>
-           <h2>Farcebook</h2>
-             <div className="username-format">{username}</div>
-             <div className='nav-item' onClick={logout}><a href='/logout'>Logout</a></div>
-           </nav>
+          <nav className='nav-bar'>
+          <h2>Farcebook</h2>
+            <div className="username-format">{username}</div>
+            <div className='nav-item' onClick={logout}><a href='/login'>Logout</a></div>
```

Comment:
Is this a typo as link href is to /login but label says Logout?

Reply:
No, this is intentional. It's a logout button, hence logout, but currently takes the user to ./logout which is a dead route when really they want to go back to ./login